README.md

```diff
@@ -131,6 +131,8 @@ List delegations created by this agent for others.
 
 ### `w3 proof add <proof.ucan>`
 
+Add a proof delegated to this agent. The proof is a CAR encoded delegation to _this_ agent. Note: use `w3 space add` to add a proof _as well as_ add the resource to your list of spaces.
```

Comment:
Aside: I could use some more educating here... My understanding is that a space proof is a specific kind of proof that delegates capabilities over the `with` which is a special did:key that has been registered as a "space" with w3s, or will be in the future.

So "some proofs grant abilities on a space and some don't"... Or do we see any `with` in a proof as potentially being a "space" and so "all proofs grant abilities over a space".

If it's the former we should tweak the wording to say more like "you probably want `space add` unless you know the delegation you received targets something other than a w3 space"

Reply:
I'm not sure I understand the difference you explained but I agree that we want to put "you probably want space add unless you know the delegation you received targets a resource other than a w3 space".

Reply:
is the `with` in a proof always gonna be considered a "space" in our world, or does the `with` sometimes point to a thing that is not a space?